servers/display_server.h

```diff
@@ -49,6 +49,12 @@ class DisplayServer : public Object {
 		return singleton;
 	}
 
+	//additional parameters,if any,that are passed in when initializing the displayserver.provide space for future expansion.
+	struct ExtraData {
```

Comment:
I think for the time being, I would add a Color and not this struct. If we want to clean this up, we should do it later in a subsequent PR but not left the code in an intermediate state.